<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>subWidget</class>
 <widget class="QWidget" name="subWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>557</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>subWidget</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>0</y>
     <width>381</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string>接收窗口</string>
   </property>
   <widget class="QTextEdit" name="recieveEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>381</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>330</y>
     <width>421</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTextEdit" name="sendEdit">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>421</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>251</width>
     <height>321</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>241</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>奇偶校验</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="bandRateBox"/>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="checkoutBox"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>串口操作</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>停止位</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="stopBitBox"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>波特率</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>数据位</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="dataBitBox"/>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="open_Button">
       <property name="text">
        <string>打开串口</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>241</width>
      <height>26</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QPushButton" name="search_Button">
       <property name="text">
        <string>串口检测</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comBox"/>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>261</y>
      <width>107</width>
      <height>54</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="clear_r_Button">
       <property name="text">
        <string>清空接受</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="show16Box">
       <property name="text">
        <string>16进制显示</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>330</y>
     <width>191</width>
     <height>171</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>131</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="send_Button">
       <property name="text">
        <string>发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="send16Box">
       <property name="text">
        <string>16进制发送</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="newlineBox">
       <property name="text">
        <string>发送新行</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout"/>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>send_Button</sender>
   <signal>clicked()</signal>
   <receiver>subWidget</receiver>
   <slot>on_clear_r_Button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>531</x>
     <y>370</y>
    </hint>
    <hint type="destinationlabel">
     <x>799</x>
     <y>516</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>search_Button</sender>
   <signal>clicked()</signal>
   <receiver>subWidget</receiver>
   <slot>on_search_Button_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>535</x>
     <y>40</y>
    </hint>
    <hint type="destinationlabel">
     <x>782</x>
     <y>199</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <signal>signal1()</signal>
  <slot>on_search_Button_clicked()</slot>
  <slot>on_open_Button_clicked()</slot>
  <slot>on_send_Button_clicked()</slot>
  <slot>on_clear_r_Button_clicked()</slot>
 </slots>
</ui>
